resources/views/admin/schedule/index.twig

```diff
@@ -6,35 +6,56 @@
 
 {% block content %}
     <div class="container">
-        <h1>{% block content_title %}{{ title }}{% endblock %}</h1>
+        <h1>
+            {% block content_title %}{{ title }}{% endblock %}
 
-        {% for message in errors|default([]) %}
-            {{ m.alert(__(message), 'danger') }}
-        {% endfor %}
-        {% for message in success|default([]) %}
-            {{ m.alert(__(message), 'success') }}
-        {% endfor %}
+            {% if schedules|default([]) %}
```

Comment:
This shows the add button only if there is at least one schedule configured. The button should always be visible.

Reply:
Whops, äh, will fix that :D